ui/templates/includes/learning_resource.html

```diff
@@ -48,7 +48,12 @@
         <div class="tile-meta">
           <span class="meta-item">{{ result.course }}</span>
           <span class="meta-item">{{ result.run }}</span>
-          <span class="meta-item"><a href="{{ result.preview_url }}" target="_blank">Preview</a></span>
+          <span class="sub-meta">
+            <span class="meta-item mi-col-1"><i class="fa fa-eye"></i> {{ result.object.xa_nr_views }}</span>
```

Comment:
These all use `result.object`, which bypasses the work done in #417. 

However, those fields are actually in `result`, so if you replace, for example, `result.object.xa_nr_views` with `result.xa_nr_views`, that should make this better.

For reference, see `search/search_indexes.py`. Any fields in `class LearningResourceIndex` should be available to you on `result`. Any calls to `result.object` will cause a database query.